src/pilot/dracclient_raid.patch

```diff
@@ -1,31 +1,55 @@
---- raid.py.orig	2019-01-23 15:57:50.655000000 +0000
-+++ raid_backup.py	2019-01-23 14:44:59.068000000 +0000
-@@ -588,14 +588,26 @@
-         """
-         return raid_controller_fqdd.startswith('RAID.')
+--- raid.py.orig	2019-02-12 00:55:47.000000000 -0500
++++ raid_new.py	2019-02-13 10:27:14.000000000 -0500
+@@ -34,6 +34,11 @@
  
--    def is_boss_controller(self, raid_controller_fqdd):
```

Comment:
It looks like this patch needs to be updated.  The patch that fixes is_boss_controller is being removed in this change, and it should not be.

Reply:
i removed is_boss_controller because currently osp13.0 is installing dracclient=1.6.0 and that repository is already having this changes merged. That's the reason i removed it

Reply:
Sorry i assumed the changes is already merged. Please ignore above comment. i will this changes in client.patch file